pom.xml

```diff
@@ -371,8 +371,8 @@
         <!-- Shaded packages -->
         <dependency>
             <groupId>com.github.thebusybiscuit</groupId>
-            <artifactId>CS-CoreLib2</artifactId>
-            <version>0.31.0</version>
+            <artifactId>cscorelib2</artifactId>
+            <version>0.32.0_6</version>
```

Comment:
Released 0.32.0, will take jitpack a minute to notice that though.
https://github.com/TheBusyBiscuit/CS-CoreLib2/releases/tag/0.32.0